### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_FilesTab_Inspect:Name is missing for the "Folder" Icon.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3992]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Chrome:Version 68.0.3440.106
Operating System:Windows10
**Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    
2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3.Click on Run or right click and select Run to run the notebook.
5.Navigate to the "Folder" Icon. 
**Actual:**
Name is missing for the "Folder" Icon.
**Expected:**
An appropriate name should be present for the "Folder" Icon.
**UserImpact:**
ScreenReader user will get no information regarding the folder present on the page.
**Attachment:**
![bugattachment4](https://user-images.githubusercontent.com/39204909/45810787-1cb7a780-bce9-11e8-9ffa-c97e6548cf56.png)
